lib/internal/errors.js

```diff
@@ -28,7 +28,6 @@ function makeNodeError(Base) {
     constructor(key, ...args) {
       super(message(key, args));
       this[kCode] = key;
-      Error.captureStackTrace(this, NodeError);
```

Comment:
oh nice... I knew that change was coming but I wasn't expecting it until 6.x for some reason. :-) ... that's a happy surprise. I withdraw the objection then!